app/assets/javascripts/mumuki_laboratory/application/faqs.js

```diff
@@ -0,0 +1,80 @@
+mumuki.faqs = class {
```

Comment:
Not a big deal, but I think I normally do `new class {` with singletons instead of `class {` and a new in some other place.